web/src/test/java/org/cbioportal/genome_nexus/web/AnnotationIntegrationTest.java

```diff
@@ -136,4 +155,86 @@ public void testLongDeletion()
         assertEquals("GRCh37", assemblyName0);
 
     }
+
+    @Test
+    public void testGenomicLocationSNVGET() {
+        String genomicLocation = "7,140453136,140453136,A,T";
+
+        Map<String, Object> response = this.fetchVariantAnnotationByGenomicLocationGET(genomicLocation);
+
+        // TODO: somehow only the POST contains hgvsg but not GET We should
+        // maybe do a more general test that checks for object equality between
+        // GET response and POST response for all types of endpoints. This is
+        // not enforced through typing because a lot of the fields are optional.
+        // assertEquals(response.get("hgvsg"), "7:g.140453136A>T");
```

Comment:
Interesting, do those fields exist for the original web service request?

Reply:
Doesn't look like it: http://grch37.rest.ensembl.org/vep/human/hgvs/7:g.140453136A%3ET?content-type=application/json&hgvs=1&hgvsg=1. I guess we are adding them in the POST requests somewhere